10-std.tex

```diff
@@ -368,6 +368,98 @@ \section{Lambda}
 }
 \end{lstlisting} 
 
+\section{Template}
+\label{std-template}
+
+Haxe comes up with a standard Template System with an easy to use syntax which is interpreted by a lightweight class called \type{haxe.Template}.
```

Comment:
Yes, that was from the old docs. Personally I think it adds quite heavy amount of stuff. I think Ill just remove _lightweight._